analyses/hello-clusters/01_perform-evaluate-clustering.Rmd

```diff
@@ -49,32 +62,443 @@ data_dir <- file.path(repository_base, "data", "current")
 module_base <- file.path(repository_base, "analyses", "hello-clusters")
 ```
 
-#### Input and output files
-
-Set paths to input and output directories and files in the chunk below.
+### Input file
 
 ```{r paths}
-# Input files
+# Path to processed SCE file for sample SCPCS000001
+input_sce_file <- file.path(data_dir, "SCPCP000001", "SCPCS000001", "SCPCL000001_processed.rds")
+```
+
+
+## Read in and prepare data
+
+To begin, we'll read in the `SingleCellExperiment` (SCE) object.
+We'll also establish a corresponding processed Seurat object from its raw counts that we'll use for some examples.
+
+```{r read data}
+# Read the SCE file
+sce <- readRDS(input_sce_file)
+```
+
+```{r create process seurat, message = FALSE}
+# Convert to a Seurat object
+seurat_obj <- CreateSeuratObject(counts = counts(sce), assay = "RNA")
 
```

Comment:
This needs to be updated with `rOpenScPCA::sce_to_seurat()` once https://github.com/AlexsLemonade/rOpenScPCA/pull/15 goes in.